felix/rules/endpoints_test.go

```diff
@@ -520,42 +578,62 @@ func endpointRulesTests(flowLogsEnabled bool, dropActionOverride string) func()
 							Rules: setEndpointMarkRules(0xd400, 0xff00),
 						},
 					})
-					Expect(renderer.WorkloadEndpointToIptablesChains(
+					actual := renderer.WorkloadEndpointToIptablesChains(
 						"cali1234",
 						epMarkMapper,
 						true,
 						tiersToSinglePolGroups([]*proto.TierInfo{{
-							Name:            "default",
-							DefaultAction:   "Pass",
-							IngressPolicies: []string{"ai", "bi"},
-							EgressPolicies:  []string{"ae", "be"},
+							Name:          "default",
+							DefaultAction: "Pass",
+							IngressPolicies: []*proto.PolicyID{
+								{Name: "ai", Kind: v3.KindNetworkPolicy},
+								{Name: "bi", Kind: v3.KindNetworkPolicy},
+							},
+							EgressPolicies: []*proto.PolicyID{
+								{Name: "ae", Kind: v3.KindNetworkPolicy},
+								{Name: "be", Kind: v3.KindNetworkPolicy},
```

Comment:
IIUC all these policies and some of the following ones should be `KindGlobalNetworkPolicy`, right?
Maybe does not matter in these tests, but worth using the correct kind anyway.